e2e/runner/run_cypress_ci.js

```diff
@@ -17,13 +27,13 @@ const startServer = async () => {
   await CypressBackend.start();
 };
 
-const runTests = async suite => {
-  printBold(`Running ${suite} Cypress Tests`);
-  await runCypress(suite, process.exit);
+const runTests = async testSuite => {
```

Comment:
small thing. This is technically incorrect since you pass `snapshot` which is defined as a mode here.

It should be `modeOrTestSuite` instead for consistentcy